.github/workflows/js_tests.yml

```diff
@@ -0,0 +1,34 @@
+name: JavaScript
+on:
+  pull_request:
+    paths:
+      - 'webpack/**'
+      - 'package.json'
+      - '.github/workflows/js_tests.yml'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version:
+          - 10
+          - 12
+    steps:
+      - name: Checkout Foreman
+        uses: actions/checkout@v2
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install npm dependencies
+        run: npm install
+      - name: Run linter
+        run: npm run lint
+      - name: Run tests
+        run: npm run test
+      - name: Publish Coveralls
+        uses: coverallsapp/github-action@master
```

Comment:
Coveralls have a special configuration when using `matrix`, see: https://github.com/marketplace/actions/coveralls-github-action#complete-parallel-job-example

Reply:
good catch, added
lets see if it works as expected